packages/gatsby/src/internal-plugins/internal-data-bridge/gatsby-node.js

```diff
@@ -109,6 +105,22 @@ exports.sourceNodes = ({ createContentDigest, actions, store }) => {
 
   createGatsbyConfigNode(state.config)
 
+  const buildTime = moment()
+    .subtract(process.uptime(), `seconds`)
+    .startOf(`second`)
+    .toJSON()
+
+  createNode({
+    id: `SiteBuildTime`,
+    parent: null,
+    children: [],
+    buildTime,
+    internal: {
+      contentDigest: createContentDigest(buildTime),
+      type: `SiteBuildTime`,
```

Comment:
Can we name it just `SiteBuild`? So that we could naturally add more build metadta there if we need it some day?

Reply:
:+1:

Reply:
That's a good idea. We might want to at some point mark the buildTime field on Site as deprecated and encourage people to access it via the metadata instead